### PR TITLE
Update with multi-module Jenkins example [TG-16186]

### PR DESCRIPTION
**Pull request by jeannielynnmoulton**

### CODE REVIEW

**Review by ThomasPerkins1123 — APPROVED** [review]

**Review by allredj — APPROVED** [review]
Can we have a link back to the pipeline documentation on the website in case someone lands on that page first?

**Comment by jeannielynnmoulton** [comment]
@ThomasPerkins1123 @allredj I've removed the old examples as we discussed in refinement today.

File: jenkins-github-maven/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    options {
+        parallelsAlwaysFailFast()
+    }
+
+    /*
+    Modify these as required
+    */
+    environment {
+       // Key for Github from credentials set up on Jenkins server
+       SSH_KEY = credentials('SSH_KEY_PIGGYMETRICS')
```
Comment (allredj) [inline jenkins-github-maven/Jenkinsfile:34]: Is it your intention to keep PiggyMetrics-related stuff in here? :pig:

File: jenkins-github-maven/scripts/common.sh [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+diffblueBotName() {
+  echo $COMMIT_BOT_NAME
+}
+diffblueBotEmail() {
+  echo $COMMIT_BOT_EMAIL
+}
+
+# This example uses github, and these are required for curl commands in updating comments.
+GITHUB_ORG="diffblue"
+GITHUB_REPO="piggymetrics"
```
Comment (allredj) [inline jenkins-github-maven/scripts/common.sh:57]: :pig:

**Review by allredj — APPROVED** [review]

File: jenkins-github-maven/Jenkinsfile (hunk 1 above)
Comment (jeannielynnmoulton) [reply jenkins-github-maven/Jenkinsfile:34]: For now, yes.

**Pull request merged**